README.md

```diff
@@ -23,6 +23,11 @@ Contributions welcome!
 - ส้มเน่า
 - เผาไทย
 - ลิเบอร่าน
+- คณะร่าน
+- เห่อหมอยคลั่งชาติฝรั่ง
+- ขุ่นพ่อง
```

Comment:
อันนี้มี ref ไหมครับ อยากรู้ context 
แต่เดี๋ยวผม merge ไปก่อน

Reply:
ref จากเพจ คณะร่าน เลยครับ
ขุ่นพ่องในที่นี้ หมายถึงปรีดี

Reply:
👀 got it

Reply:
แต่ผมคิดว่าน่าจะเติมอะไรให้มันมี context เพิ่มได้ เพราะถ้าเห็นคำนี้เฉยๆ มันสื่อใจความได้ทั้งสองฝั่งเลยครับ 
ถ้าคำนี้ไม่เป็นที่พูดถึงกันอีกผมขออนุญาตเอาออกทีหลังนะครับ 🙏